JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -96,7 +97,7 @@ private fun InterestsScreenBody(
     Column {
         TabRow(items = sectionTitles, selectedIndex = currentSection.ordinal) { index, text ->
             Tab(
-                text = text,
+                text = { Text(text) } ,
```

Comment:
Maybe rename the `text` lambda param to `title` or similar for clarity i.e. less than 3 instances of the work "text" on this line!

Reply:
Text("text").text { Text(text) }